README.adoc

```diff
@@ -170,6 +166,29 @@ config.servicemanager.http_proxy_password = <Proxy user password>
 
 When these settings are applied, they are passed through to the Docker and OpenShift services. In an unauthenticated proxy environment, the `http_proxy_user` and `http_proxy_password` configurations can be omitted.
 
+=== Configuring Services
+The vagrant-service-manager helps you configure the service of your choice:
+
+. Enable the desired service(s) in the ADB Vagrantfile as:
++
+`config.servicemanager.services = 'openshift'`
++
+[NOTE]
+====
+- Docker is the default service for the Atomic Developer Bundle and does not require any configuration to ensure it is started whereas, the Red Hat Enterprise Linux Container Development Kit, which is based on ADB, automatically starts OpenShift. +
+- You can use a comma-separated list to enable multiple services. For instance: docker, openshift.
+====
+
+. Enable the relevant option for the services you have selected in the Vagrantfile. For example, specific versions of OpenShift can be set using the following variables:
++
+
+- `config.servicemanager.openshift_docker_registry = "docker.io"` - Specifies the registry from where the service should be pulled.
```

Comment:
> does this registry setting only apply for the OpenShift image? 

yes that's right.

> I would not think so, right? Is this option just a away to add an additional registry?

No to add another registry to docker we have different script https://github.com/projectatomic/adb-utils/blob/master/utils/add_insecure_registry.

this option is for "where OpenShift image pulled from".